Packages/KeyAppKit/Sources/KeyAppNetwork/Endpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Endpoint.swift
+//  p2p_wallet
+//
+//  Created by Ivan on 28.04.2022.
+//
+
+import Foundation
+
+public protocol Endpoint {
```

Comment:
Why not to use URLRequest?

Reply:
Good idea, but it's better to have a wrapper for URLRequest, because Apple changes naming pretty frequently. With a wrapper it's not going to be problematic

Reply:
I think URLRequest is not convenient to use, so wrapper is a good option.